mpt/impl/imap-mailbox/external-james/src/test/resources/org/apache/james/imap/scripts/ValidateDeployment.test

```diff
@@ -236,6 +236,7 @@ S: A10 OK FETCH completed\.
 
 C: A11 LIST "" "*"
 SUB {
+S: \* LIST \(\\HasNoChildren\) "\." "fetchTest"
```

Comment:
Does this means you enforce the order the test methods should be runned in?

Reply:
sadly, do you have any idea about that?

Reply:
Can we delete the fetchTest mailbox at the end of the tests, at the very least?